src/Slider.js

```diff
@@ -165,10 +198,17 @@ var Slider = React.createClass({
     });
   },
   componentWillReceiveProps: function(nextProps) {
-    var oldValue = this.props.value;
-    var newValue = nextProps.value;
+    var props    = this.props,
+        oldValue = props.value,
+        newValue = nextProps.value;
+
```

Comment:
nit: I prefer not grouping individual declarations (use destructuring if needed).